module/cyphersystem.js

```diff
@@ -836,112 +836,112 @@ function allInOneRollDialog(actor, pool, skill, assets, effort1, effort2, additi
     if (pool == "Might" && cost > actor.data.data.pools.might.value || pool == "Speed" && cost > actor.data.data.pools.speed.value || pool == "Intellect" && cost > actor.data.data.pools.intellect.value) {
       additionalSteps = Math.abs(additionalSteps);
       allInOneRollDialog(actor, pool, skillRating, assets, effort1, effort2, additionalCost, additionalSteps, stepModifier);
-      return ui.notifications.notify(`You don’t have enough ${pool} points.`);
+      return ui.notifications.notify(`${game.i18n.localize("CYPHERSYSTEM.NotEnoughPoint")} ${pool}.`);
     }
 
     if (additionalSteps == 1 || additionalSteps == -1) steps = " step";
 
-    if (cost == 1) points = " point";
+    if (cost == 1) points = " " + game.i18n.localize("CYPHERSYSTEM.Point");
 
-    if (title == "") title = pool + " Roll";
+    if (title == "") title = pool + " " + game.i18n.localize("CYPHERSYSTEM.Roll");
 
-    if (effort1 == 1) rollEffort = " level";
+    if (effort1 == 1) rollEffort = " " + game.i18n.localize("CYPHERSYSTEM.Level").toLowerCase();
 
-    if (effort2 == 1) otherEffort = " level";
+    if (effort2 == 1) otherEffort = " " + game.i18n.localize("CYPHERSYSTEM.Level").toLowerCase();
 
-    if (damageEffort == 1) damageEffortLevel = " level";
+    if (damageEffort == 1) damageEffortLevel = " " + game.i18n.localize("CYPHERSYSTEM.Level").toLowerCase();
```

Comment:
Avoid `toLowerCase()`. German needs this capitalized.

Reply:
In this case every language will have it capitalised

Reply:
So the `toLowerCase()` is there by mistake?

Reply:
Wait, in English it’s not capitalized.

Reply:
Not sure how to address this one tbh. As it seems very few languages need it capitalised.
So either everyone will have it cap or not.